CHANGELOG.md

```diff
@@ -116,15 +116,15 @@
   [#10044](https://github.com/Kong/kong/pull/10044)
 - **OpenTelemetry**: Fix non-compliance to specification for `http.uri` in spans. The field should be full HTTP URI.
   [#10036](https://github.com/Kong/kong/pull/10036)
+- **OAuth2**: `refresh_token_ttl` is now limited between `0` and `100000000` by schema validator. Previously numbers that are too large causes requests to fail.
+  [#10068](https://github.com/Kong/kong/pull/10068)
 
 ### Changed
 
 #### Hybrid Mode
 
 - Revert the removal of WebSocket protocol support for configuration sync,
   and disable the wRPC protocol.
-  [#9921](https://github.com/Kong/kong/pull/9921)
```

Comment:
Why did we remove this PR link? any explain?

Reply:
I don't know. Maybe a mistake when solving conflict. Let's add it back.